src/visbook/visbook.api.ts

```diff
@@ -0,0 +1,60 @@
+import { AccommodationAvailability } from './visbook.interface';
+
```

Comment:
Might be nice to have a link to the documentation that you found as a comment for this module. Will make changes in the future easier :)

Reply:
Good idea, done!